tests/src/python/test_qgsdelimitedtextprovider.py

```diff
@@ -878,6 +879,8 @@ def test_044_ZM(self):
         assert vl.isValid(), f"{basetestfile} is invalid"
         assert vl.wkbType() == QgsWkbTypes.PointZM, "wrong wkb type, should be PointZM"
         assert vl.getFeature(2).geometry().asWkt() == "PointZM (-71.12300000000000466 78.23000000000000398 1 2)", "wrong PointZM geometry"
+        self.assertEqual(vl.extent(), QgsRectangle(-71.12300000000000466, 66.32999999999999829, -65.31999999999999318, 78.29999999999999716))
```

Comment:
```suggestion
        self.assertEqual(vl.extent(), QgsRectangle(-71.12300000000000466, 66.32999999999999829, -65.31999999999999318, 78.29999999999999716))
```

Theses tests will be flaky on different machines due to precision. Hope, you can use https://github.com/qgis/QGIS/pull/55648

Reply:
Indeed. I have switched to `self.assertAlmostEqual`  on each coordinate in the meantime. This is how the other providers test the extent.